readme.md

```diff
@@ -38,6 +35,10 @@ Check to see if the server started successfully by visiting http://localhost:399
 
 ## Local Development
 
+### Requirements
+
+An API client is generated automatically with [OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator). This has a dependency on [Java RE](https://www.java.com/en/download/).
```

Comment:
In the develop branch, there root proj no longer a direct dependency on the client lib, so the JRE so no longer necessary unless specifically trying to re-build the client lib. Should we keep this change in?